src/tools/tidy/src/ext_tool_checks.rs

```diff
@@ -224,6 +248,18 @@ fn check_impl(
         shellcheck_runner(&merge_args(&cfg_args, &file_args_shc))?;
     }
 
+    if js_lint {
+        rustdoc_js::lint(librustdoc_path, tools_path, src_path)?;
```

Comment:
Didn't we say that the checks would be run if the JS files were modified too? Having them never run unless an extra arguments is passed is very inconvenient.